tasks/config/simplemocha.js

```diff
@@ -1,3 +1,11 @@
+var wrap = require('lodash').wrap;
+var Mocha = require('mocha');
+
+Mocha.prototype.run = wrap(Mocha.prototype.run, function (orig) {
+  require('../../test/mocha_setup');
+  orig.call(this);
+});
```

Comment:
Messing with Mocha's prototype doesn't feel good, even if it is a really simple wrapping. I don't have another recommendation atm though because I don't understand the auto-releases-sinon stuff and why it needs to be set up the way it is.